test/pylib/manager_client.py

```diff
@@ -288,3 +291,8 @@ async def _not_sees_another_server():
             if not other_ip in alive_nodes:
                 return True
         await wait_for(_not_sees_another_server, time() + interval, period=.5)
+
+    async def server_get_log_filename(self, server_id: ServerNum) -> ScyllaLogFile:
```

Comment:
The function is not returning just the file name, it's returning a `ScyllaLogFile`, let's give a better name to the function, maybe just `server_open_log`?

BTW. is there any resource we need to close after obtaining `ScyllaLogFile` and doing whatever we need to?

Reply:
> BTW. is there any resource we need to close after obtaining ScyllaLogFile and doing whatever we need to?

No, we open a log file for each `ScyllaLogFile`'s method call